docs/index.md

```diff
@@ -1,3 +1,17 @@
+# Charmed Wazuh Dashboard Documentation
+The [Wazuh Dashboard](https://documentation.wazuh.com/current/user-manual/wazuh-dashboard/index.html)
+is a component for analyzing and visualizing security data data based on [OpenSearch](http://opensearch.org/). Consequentily,
+this charm is a Charmed OpenSearch Dashboards fork.
 
+For the complete documentation, refer to [upstream](https://charmhub.io/opensearch-dashboards).
```

Comment:
You may also want to include a link to the Wazuh Server doc set

Reply:
Where and how?

Reply:
Updated